addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1631,15 +1631,22 @@ body.editor_enable.editor_has_snippets {
             float: none;
             width: 100%;
             padding: 0;
-            margin: 0;
+            margin: $o-we-sidebar-content-field-spacing 0 $o-we-sidebar-content-field-spacing * .5 0;
```

Comment:
Add parenthesis around the operation for multiple values properties, otherwise it is not readable